packages/carbon-web-components/src/components/data-table/table-cell.ts

```diff
@@ -27,10 +38,25 @@ class BXTableCell extends LitElement {
   }
 
   render() {
-    return html` <slot></slot> `;
+    const { disabled, textContent } = this;
+
+    const statusText = ['Starting', 'Active', 'Disabled'];
+    const isStatus = statusText.some((word) => textContent?.includes(word));
+    const isDisabled = textContent!.includes('Disabled');
+    const cellClasses = classMap({
+      [`${prefix}--link`]: isStatus,
+      [`${prefix}--link--disabled`]: disabled || isDisabled,
+    });
```

Comment:
i'm not seeing this in react https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/DataTable/TableCell.tsx.... might help with the styles rich pointed out